.github/workflows/ci.yml

```diff
@@ -77,6 +77,13 @@ jobs:
                   make
                   sudo make install
 
+            - name: Install Pallene Tracer
+              run: |
+                  git clone https://github.com/pallene-lang/pallene-tracer
```

Comment:
We should create version tags in that repo, and import a particular version. Otherwise, Pallene builds might break if there new commits over there.

Reply:
Let's do that when Pallene Tracer is stabilized.